File: Src/Getpar/testpar.c
#include	<stdio.h>
int nx	=13;
float dx	=0.01;
float dy;
char title[40]	="Nothing here";
char options[128];
int map	=1;
int boo	=1;
int vec[10];
main(ac,av,ae)
int ac; char **av, **ae;
   {
	int i, n, nopt;
	char *optlist[10];
	setpar(ac,av,ae);
	getpar("nx","d",&nx);
	getpar("map","b",&map);
	getpar("dx","f",&dx);
	getpar("title","s",title);
	getpar("options","s",options);
	getpar("boo","b",&boo);
	mstpar("dy","f",&dy);
	/*
	nopt= getlist(options,"s",optlist,10);
	*/
	n= getpar("vec","vd[6]",vec);
	fprintf(stderr,"n= %2d vec=",n);
	for(i=0; i<n; i++) fprintf(stderr,"%3d, ",vec[i]);
	fprintf(stderr,"\n");
	endpar();
	fprintf(stderr,"programe now executes\n");
	fprintf(stderr,"nx=%d dx=%f map=%d title=%s\n",nx,dx,map,title);
	fprintf(stderr,"boo=%d\n",boo);
	fprintf(stderr,"options= %s\n",options);
	/*
	for(i=0; i<nopt; i++)
		fprintf(stderr,"\t%s\n",optlist[i]);
	*/
   }

File: Src/Getpar/testpar.f

	integer boo
	real dy
	character*40 title, options
	data nx/13/, dx/0.01/, map/1/, boo/1/
	data title/'Nothing here'/
	call setpar()
	call getpar('nx','d',nx)
	call getpar('map','b',map)
	call getpar('dx','f',dx)
	call getpar('title','s',title)
	call getpar('options','s',options)
	call getpar('boo','b',boo)
	call mstpar('dy','f',dy)
	call endpar()
	write(6,100) nx, map, dx, boo, dy
100	format(2x,'nx=',i5,' map=',i3,' dx=',f6.2,' boo=',i3,' dy=',f6.2)
	call pr('title',title)
	call pr('options',options)
c	write(6,101) title, options
c101	format(2x,'title=',40a1,'  options=',40a1)
	stop
	end
